### PR TITLE
feat: Improve blockstore deployment

### PR DESCRIPTION
**Pull request by arbrandes**
Allow setting more Django service variables.

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @arbrandes! I've created [OSPR-5932](https://openedx.atlassian.net/browse/OSPR-5932) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: playbooks/roles/blockstore/meta/main.yml [hunk 1]
```diff
@@ -14,20 +14,22 @@ dependencies:
...
     edx_django_service_max_webserver_upload: 10
     edx_django_service_nginx_port: '{{ BLOCKSTORE_NGINX_PORT }}'
+    edx_django_service_nginx_read_timeout: '{{ BLOCKSTORE_REQUEST_TIMEOUT }}'
     edx_django_service_ssl_nginx_port: '{{ BLOCKSTORE_SSL_NGINX_PORT }}'
     edx_django_service_default_db_name: '{{ BLOCKSTORE_DEFAULT_DB_NAME }}'
     edx_django_service_db_user: '{{ BLOCKSTORE_DATABASE_USER }}'
     edx_django_service_db_password: '{{ BLOCKSTORE_DATABASE_PASSWORD }}'
     edx_django_service_default_db_host: '{{ BLOCKSTORE_DATABASE_HOST }}'
     edx_django_service_default_db_atomic_requests: true
-    edx_django_service_default_db_conn_max_age: '{{ BLOCKSTORE_DATABASE_CONN_MAX_AGE }}'
```
Comment (arbrandes) [inline playbooks/roles/blockstore/meta/main.yml:30]: Removing this line looks like the opposite of "improvement", but it's on purpose.  This causes a type conversion error on the Ansible run.  Most roles don't use this, and I figure the error is why.

**Comment by natabene** [comment]
@arbrandes Thank you for your contribution. Is this ready for a review?

**Comment by arbrandes** [comment]
@natabene, it is, thanks for asking!

**Review by nadeemshahzad — APPROVED** [review]

**Comment by arbrandes** [comment]
Thanks for merging, @nadeemshahzad! 👍🏼

**Pull request merged**